### PR TITLE
refactor: LT-1077 - Add KeyValueChoiceField for serializers

### PR DESCRIPTION
**Pull request by jfaracik**
* Added KeyValueChoiceField - returns `{'key': 'value'}` instead of just key in serializers
* Cleaned up some tests
* Added last updated field for pick list items

### CODE REVIEW

File: picklists/tests/tests_update.py [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
 
-        url = reverse('picklist_items:picklist_item', kwargs={'pk': picklist_item.id})
-        response = self.client.put(url, data, **self.gov_headers)
-        response_data = json.loads(response.content)
+        response = self.client.put(self.url, data, **self.gov_headers)
+        response_data = response.json()
 
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(response_data['picklist_item']['status'], PickListStatus.DEACTIVATE)
+        self.assertEqual(response_data['picklist_item']['status'], {'key': PickListStatus.DEACTIVATE,
```
[the remark this answers is not in the document]
Comment (jfaracik) [reply picklists/tests/tests_update.py]: In this case the key and value of PickListStatus.DEACTIVATE are the same, however that's not always the case, eg: The key for STANDARD_ADVICE is standard_advice but the value is Standard Advice.

[the remark this answers is not in the document]
Comment (jfaracik) [reply picklists/tests/tests_update.py]: I'm going to change the key of DEACTIVATE to be lower case, so I'll update the test accordingly.

File: picklists/models.py [hunk 2]
```diff
@@ -14,3 +14,4 @@ class PicklistItem(models.Model):
     text = models.TextField(blank=False, max_length=5000)
     type = models.CharField(choices=PicklistType.choices, max_length=50)
     status = models.CharField(choices=PickListStatus.choices, default=PickListStatus.ACTIVATE, max_length=50)
+    last_updated = models.DateTimeField(auto_now=True, blank=True)
```
Comment (5pence) [inline picklists/models.py]: please change to last_updated_at or something with _at at the end in order to show it is a date time field

File: picklists/serializers.py [hunk 3]
```diff
@@ -30,5 +35,5 @@ class Meta:
                   'text',
                   'type',
                   'status',
-                  'team_name')
-
+                  'team_name',
+                  'last_modified_at')
```
Comment (andytaylorkainos) [inline picklists/serializers.py]: end tuple with a `,` please

**Pull request merged**